tests/test_harvest_products.py

```diff
@@ -54,11 +74,13 @@ class MockJob:
     }
     s3_stubber.add_response(method='copy_object', expected_params=params, service_response={})
 
-    files = harvest_products.harvest(product, MockJob())
+    mock_harvest_image = 'https://foo.com/file.png'
+    with mock.patch('harvest_products.harvest_image', lambda x, y, z: mock_harvest_image):
```

Comment:
might read better without the local variable, just `lambda x, y, z: 'https://foo.com/file.png'`, it's fine like this, though

Reply:
would mean asserting against literal strings at the end of the test, which I think we slightly prefer